units/en/unitbonus1/train.mdx

```diff
@@ -0,0 +1,265 @@
+# Let's train and play with Huggy 🐶 [[train-play]]
+
+## Let's train Huggy 🐶
+
+**To start to train Huggy, click on Open In Colab button** 👇 :
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)]()
+
+
+# Bonus Unit 1: Let's train Huggy the Dog 🐶 to fetch a stick
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit2/thumbnail.png" alt="Bonus Unit 1Thumbnail">
+
+In this notebook, we'll reinforce what we learn in the first Unit by **teaching Huggy the Dog to fetch the stick and then play with it directly in your browser**
+
+⬇️ Here is an example of what **you will achieve at the end of the unit.** ⬇️ (launch ▶ to see)
+
+```python
+%%html
+<video controls autoplay><source src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/notebooks/unit-bonus1/huggy.mp4" type="video/mp4"></video>
+```
+
+### The environment 🎮
+
+- Huggy the Dog, an environment created by [Thomas Simonini](https://twitter.com/ThomasSimonini) based on [Puppo The Corgi](https://blog.unity.com/technology/puppo-the-corgi-cuteness-overload-with-the-unity-ml-agents-toolkit)
+
+### The library used 📚
+
+- [MLAgents (Hugging Face version)](https://github.com/huggingface/ml-agents)
+
+We're constantly trying to improve our tutorials, so **if you find some issues in this notebook**, please [open an issue on the Github Repo](https://github.com/huggingface/deep-rl-class/issues).
+
+## Objectives of this notebook 🏆
+
+At the end of the notebook, you will:
+
+- Understand **the state space, action space and reward function used to train Huggy**.
+- **Train your own Huggy** to fetch the stick.
+- Be able to play **with your trained Huggy directly in your browser**.
+
+
+
+
+## This notebook is from Deep Reinforcement Learning Course
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/notebooks/deep-rl-course-illustration.jpg" alt="Deep RL Course illustration"/>
+
+In this free course, you will:
+
+- 📖 Study Deep Reinforcement Learning in **theory and practice**.
+- 🧑‍💻 Learn to **use famous Deep RL libraries** such as Stable Baselines3, RL Baselines3 Zoo, CleanRL and Sample Factory 2.0.
+- 🤖 Train **agents in unique environments**
+
+And more check 📚 the syllabus 👉 https://simoninithomas.github.io/deep-rl-course
+
+Don’t forget to **<a href="http://eepurl.com/ic5ZUD">sign up to the course</a>** (we are collecting your email to be able to **send you the links when each Unit is published and give you information about the challenges and updates).**
```

Comment:
As the entry point is now directly from the course, I would consider removing this or make it much shorter (e.g. just have a sentence, not paragraphs)

Reply:
I reduced it